src/xserver_uv.cpp

```diff
@@ -15,6 +15,7 @@
 #include "xeus-uv/xhook_base.hpp"
 
 #include "xeus-zmq/xcontrol_default_runner.hpp"
+#include "xeus-zmq/xshell_default_runner.hpp"
 #include "xeus-zmq/xserver_zmq_split.hpp"
 
 #include "xeus/xserver.hpp"
```

Comment:
You need to include `xeus-uv/xserver_uv.hpp` instead of `xeus/xserver.hpp`. Otherwise, the linker considers that the symbols defined in this file are unrelated to that declared in the header file, leading to a link error when trying to use them in a downstream lib / binary.